plaso/parsers/nids_fast.py

```diff
@@ -0,0 +1,304 @@
+# -*- coding: utf-8 -*-
+"""Parser for NIDS alert data in the so-called fast-log format,
+
+Fast.log format is a single line log format as shown below. There exist two
+variants:
+
+  Snort3:
+    Month/Day-Hour:Minute:Second:FractionOfSeconds [**] [sid] "string" [**] \
+    [Classification: string] [Priority : int] {protocol} SOURCE_IP:SOURCE_PORT \
+    -> DESTINATION_IP:DESTINATION_PORT
+
+  Suricata:
+    Year/Month/Day-Hour:Minute:Second:FractionOfSeconds [**] [sid] string [**] \
+    [Classification: string] [Priority : int] {protocol} SOURCE_IP:SOURCE_PORT \
+    -> DESTINATION_IP:DESTINATION_PORT
+
+More information on the format specification of Suricata can be found here:
+  https://suricata.readthedocs.io/en/suricata-6.0.0/configuration/suricata-yaml.html#line-based-alerts-log-fast-log
+
+"""
+
+import re
+
+import pyparsing
+from dfdatetime import time_elements as dfdatetime_time_elements
+from plaso.containers import events, time_events
+from plaso.lib import definitions, errors
+from plaso.parsers import manager
+from plaso.parsers import text_parser as tp
+
+
+class NIDSFastAlertEventData(events.EventData):
+    """Fast log event data.
+
+    Attributes:
+      sid (str): identifier of the rule that triggered the alert,
+      message (str): message associated with the alert,
+      priority (int): optional integer ranging from 1 (high) to 4 (very low),
+      classification (str): optional classification of the alert,
+      source_ip (str): source IPv4- or IPv6-address,
+      source_port (int): optional source port number,
+      destination_ip (str): destination IPv4- or IPv6-address,
+      destination_port (int): optional destination port number,
+    """
+
+    DATA_TYPE = "nids:fast:alert"
+
+    def __init__(self):
+        """Initializes event data."""
+        super(NIDSFastAlertEventData, self).__init__(data_type=self.DATA_TYPE)
+        self.sid = None
+        self.message = None
+        self.priority = None
+        self.classification = None
+        self.protocol = None
+        self.source_ip = None
+        self.source_port = None
+        self.destination_ip = None
+        self.destination_port = None
+
+
+class NIDSFastParser(tp.PyparsingSingleLineTextParser):
+    """NIDS fast.log parser."""
```

Comment:
Plaso uses 2 space indentation

Reply:
Addressed in 66015c33f8078ac2351cb1706f3111ec86734de8.